README.md

```diff
@@ -5,7 +5,8 @@
 ## Usage
 
 ```tsx
-import { OrderManagerProvider, useOrderManager } from 'vtex.order-manager/OrderManager'
```

Comment:
The previous way will work now

Reply:
True. I reverted this change in the readme.